format/Schema.fbs

```diff
@@ -59,24 +59,24 @@ table FixedSizeList {
 
 /// A Map is a logical nested type that is represented as
 ///
-/// List<entry: Struct<key: K, value: V>>
+/// List<entries: Struct<key: K, value: V>>
 ///
 /// In this layout, the keys and values are each respectively contiguous. We do
 /// not constrain the key and value types, so the application is responsible
 /// for ensuring that the keys are hashable and unique. Whether the keys are sorted
-/// may be set in the metadata for this field
+/// may be set in the metadata for this field.
 ///
-/// In a Field with Map type, the Field has a child Struct field, which then
+/// In a field with Map type, the field has a child Struct field, which then
 /// has two children: key type and the second the value type. The names of the
-/// child fields may be respectively "entry", "key", and "value", but this is
-/// not enforced
+/// child fields may be respectively "entries", "key", and "value", but this is
```

Comment:
is it worth bolding "may" and not enforced to avoid confusion.  Should there be a JIRA follow-up to update integration tests to verify this isn't required?

Reply:
I think it's clear enough without adding formatting. Yes, it would be good to add test to verify this, I'll make a JIRA.

Reply:
Made https://issues.apache.org/jira/browse/ARROW-7173 to add tests